Common.sln

```diff
@@ -52,6 +52,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{D2D23857
 		build\repo.props = build\repo.props
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Extensions.Certificates.Generation.Tests", "test\Microsoft.Extensions.Certificates.Generation.Tests\Microsoft.Extensions.Certificates.Generation.Tests.csproj", "{02774467-AA73-4283-A46D-635F1FB0B90D}"
```

Comment:
Nit: can you condense this into the existing test project, Microsoft.Extensions.Internal.Test? Creating an new test assembly for one test class seems like a overkill...plus it makes tests a lot slower as test assemblies can't run in parallel.

Reply:
This is netcoreapp2.0 only, so its kind of painful to be moving it to the internal project. Build time is not a problem in common and I rather keep it simple

Reply:
`#if NETCOREAPP2_0`